### ISSUE TITLE
Can't resume syncing folder in reverse mode after faliure

### ISSUE THREAD

**Issue by ruippeixotog** [google/adb-sync#19]
I tried to use `adb-sync` to sync the contents of `/sdcard` to a local directory. However, I had to terminate the command before it finished.

I expected to resume the operation later, just like `rsync` and other file syncing tools, but when I tried the following message appeared:

```
❯ adb-sync -R /sdcard .
Sync: local ./sdcard, remote /sdcard
Scanning and diffing...
Pull-Delete-Conflicting: ./sdcard
Would have to replace to do this. Use --force to allow this.
Total: 0 KB/s (0 bytes in 0.432s)
```

The error is too vague to understand what's going on. Is something like this not allowed?

**Comment by divVerent** [comment]
Try /sdcard/. as source. /sdcard is a symlink after all.

**Comment by ruippeixotog** [comment]
Yeah, I remembered that later and ended up syncing with `/storage/emulated/0` instead. I forgot to update this issue, sorry. Thanks anyway for the help :)

Anyway, `adb-sync -R /sdcard <local_path>` worked the first time (when I didn't have a local copy at all) and was syncing the folder as I expected initially. While I understand why it failed when I tried the second time, shouldn't the behavior be the same in the two cases, for consistency? Maybe `adb-sync` should always refuse to sync a symlink?
